### DISCUSSION TITLE
Providing custom Google Map URL to the component

### DISCUSSION THREAD

**Discussion by asaivishal** [JustFly1984/react-google-maps-api#2038] [Ideas]
HI,

We have a requirement to provide different Google Map URL to the component based on one specific country (CN - China) and the google map url for that is different.  With `react-google-maps@9.4.0` we were able to provide different url using `googleMapURL` prop in `withScriptJs` HOC.

Using `@react-google-maps/api` there doesn't seem to have any way to customise URL because it's using `makeLoadScriptUrl` utility function to create google map URL and it's using static URL value `return https://maps.googleapis.com/maps/api/js?${params.join('&')}`.

Is there any way to provide custom URL or there should be custom prop or params to provide different value of base google maps url?

**Comment by asaivishal** [comment]
@JustFly1984 any idea how to provide custom URL?

**Reply by hsource** [reply]
I think you can submit a patch to add this as a prop that could be passed in! Perhaps a `host` or a full URL

**Reply by JustFly1984** [reply]
There is a multiple ways to load the script, including official google package in dependencies.

You can address your issue in official google repository, or just copy any of the load script component from this library into your own codebase and edit it as you like.

As somebody posted above, you can make a PR with your Chinese supported Component into this repo to give back to community.

**Comment by goacgras** [comment]
@asaivishal Hey did you find a way to be accessible in china?
